app/src/main/java/org/greenstand/android/TreeTracker/utilities/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package org.greenstand.android.TreeTracker.utilities
+
+object Constants {
+    const val TREE_TRACKER_URL = "http://www.treetracker.org"
```

Comment:
This link doesn't seem to work for me. You can use "https://map.treetracker.org/"